pocs/focuser/focuser.py

```diff
@@ -490,14 +463,13 @@ def _autofocus(self,
 
         final_focus = self.move_to(best_focus)
 
+        final_fn = "{}_{}.{}".format(final_focus, "final", self._camera.file_extension)
```

Comment:
Should this include the `focus_type`? I don't seem to see it set anywhere.

Reply:
That's a good idea. It should. I'll put that in.